editorconfig-lint-api/src/main/java/org/ec4j/lint/api/Constants.java

```diff
@@ -188,7 +188,12 @@
             // fonts
             "**/*.eot", //
             "**/*.ttf", //
-            "**/*.woff" //
+            "**/*.woff", //
+
+            // Java archives
+            "**/*.ear", //
+            "**/*.sar", //
+            "**/*.war" //
```

Comment:
These are are archives much like jar. Could you plz. move them up, after jar?

Reply:
OK, how about now?